include_core/omrport.h

```diff
@@ -297,6 +346,18 @@ typedef struct J9Permission {
 	uint32_t : 26; /* future use */
 } J9Permission;
 
+typedef struct OMRPortShsemStatistic {
+        uintptr_t semid;
+        uintptr_t ouid;
+        uintptr_t ogid;
+        uintptr_t cuid;
+	uintptr_t cgid;
+	int64_t lastOpTime;
+	int64_t lastChangeTime;
+	int32_t nsems;
+        J9Permission perm;
+} OMRPortShsemStatistic;
```

Comment:
Another example: this struct is using a mix of tabs and spaces for indentation, when it should be all tabs.

Reply:
Thank you for your comment. Modified in latest commit.